ui/app/AppLayouts/Communities/views/CommunityColumnView.qml

```diff
@@ -406,26 +406,29 @@ Item {
             anchors.topMargin: Theme.padding
             spacing: Theme.bigPadding
 
+            Settings {
```

Comment:
It's quite a mess currently in the codebase now regarding where we put `Settings` instances. The first step to manage it a bit better would be to push those instances up in the hierarchy. It also improves testability and isolation when settings are higher.

Reply:
In this case we don't really need to; it's a simple local setting, close it, write the bool somewhere. Actually even for storybook or tests, as we use a different instance of the settings file then:

```cpp
QStandardPaths::setTestModeEnabled(true);
```

So the settings storage becomes a temporary file